docs/Create-Packages/Creating-a-Package.md

```diff
@@ -236,22 +236,20 @@ When using a convention-based working directory, on the other hand, NuGet assume
 
 Such a folder structure also allows you to easily include files that might not be part of a project at all, including:
 
-- Content and source code that should be injected into the target project.
-- PowerShell scripts (packages used in NuGet 2.x can include installation scripts as well, which is not supported in NuGet 3.x and later).
-- Transformations to existing configuration and source code files in a project.
+- Content and source code that should be available for reference across projects through global packages folder.
+- PowerShell scripts only initialization script init.ps1 (packages used with NuGet packages.config format can include installation scripts as well, which is not supported in NuGet project.json and packageReference formats).
```

Comment:
Should it be **P**ackageReference? However it is written it should be consistent everywhere.

Reply:
hmm make sense!